core/src/main/java/com/decibeltx/studytracker/core/repository/EntryTemplateRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.decibeltx.studytracker.core.repository;
+
+import com.decibeltx.studytracker.core.model.EntryTemplate;
+import org.springframework.data.mongodb.repository.MongoRepository;
+
+import java.util.Date;
+
+public interface EntryTemplateRepository extends MongoRepository<EntryTemplate, String> {
```

Comment:
Again, if we are not going to display this on the main page, let's not add the extra queries.